server/util/index.ts

```diff
@@ -0,0 +1,14 @@
+import { parseAddress } from 'addresser';
+
+export function createKeys(property: string, values: string[]) {
+  return values.map((v) => ({ [property]: v }));
+}
+
+export function formatAddress(address: string): string {
+  let addressString = address;
+  if (!address.includes(',')) {
```

Comment:
Also, what if they enter something like "Apartment Name, Apartment #" that has ',' but not the rest of the address?

Reply:
good point, ill add a better validation check